src/drop-down-icon.jsx

```diff
@@ -112,7 +112,7 @@ const DropDownIcon = React.createClass({
           <div onTouchTap={this._onControlClick}>
               <FontIcon
                 className={iconClassName}
-                style={this.prepareStyles(iconStyle)}>{this.props.iconLigature}</FontIcon>
+                style={iconStyle}>{this.props.iconLigature}</FontIcon>
```

Comment:
`<FontIcon>` (and all material components) use `prepareStyles` when passing styles to dom elements. So we shouldn't prefix anything passed to another component.